author/views.py

```diff
@@ -1,3 +1,23 @@
-from django.shortcuts import render
+from django.views import generic
+from .models import Author
+from django.core.paginator import Paginator
 
-# Create your views here.
+class BloggerDetailView(generic.DetailView):
+    queryset = Author.objects.all()
+    template_name = "author/blogger_detail.html"
+    context_object_name = "author"
+    
+    def get_queryset(self):
+        return self.queryset.filter(id=self.kwargs['pk'])
```

Comment:
Here you have to use 

queryset = super().get_queryset()

because this will execute query 2 times so that it will impact on performance